requirements_test.txt

```diff
@@ -1,2 +0,0 @@
-mock==2.0.0
```

Comment:
How would you feel about moving these into a "testing" `extras_require` section in the setup.py, like [this one](https://github.com/cfpb/wagtail-sharing/blob/master/setup.py#L30)?

This would allow users to write `pip install -e '.[testing]'` to have testing requirements installed if they want to, while still making `pip install .` work without installing them.